gimme

```diff
@@ -81,13 +81,32 @@ _do_curl() {
 	exit 1
 }
 
+# _sha256sum "file"
+_sha256sum() {
+	if command -v sha256sum &>/dev/null; then
+		sha256sum "$@"
+	else
```

Comment:
Suggest: `elif command -v gsha256sum &>/dev/null; then`  + `gsha256sum "$@"`

This `g` prefix is fairly common for GNU utils, and is used, amongst others, by Homebrew when installing `coreutils`.